### DISCUSSION TITLE
Problem for install plugin config_officer

### DISCUSSION THREAD

**Discussion by RenatoPereira91** [netbox-community/netbox#9548] [General]
Hi,

I'm the problem for install the plugin 'config_officer' (https://github.com/artyomovs/netbox-plugin-config-officer).

I tried using docker but I get the wrong below:
ImportError: cannot import name 'PrimaryModelFilterSet' from 'netbox.filtersets' (/opt/netbox/netbox/netbox/filtersets.py)

I tried using pip install netbox-plugin-config-officer, it installed but after configure and restart service I get the mensage:

<class 'django.urls.exceptions.NoReverseMatch'>

'config_officer' is not a registered namespace inside 'plugins'

Python version: 3.8.10
NetBox version: 3.2.5-dev

**Comment by bogdancordos** [comment]
Hi, the plugin is not yet compatible with NetBox 3.2.x.

**Reply by RenatoPereira91** [reply]
Hi @bogdancordos 

It is good plugin because it do the complice, do you know other plugin that can run complice config?

**Reply by bogdancordos** [reply]
Hi @RenatoPereira91 , I'm not aware of another plugin that does the same thing, we can only hope that the plugin developer will update its code.
